R/helper_add_nflscrapr_mutations.R

```diff
@@ -666,8 +666,8 @@ make_model_mutations <- function(pbp) {
 
 
 fix_scrambles <- function(pbp) {
-  # skip below code if 2005 is not in the data
-  if (!2005 %in% pbp$season) return(pbp)
+  # skip below code if <= 2005 is not in the data
+  if (min(pbp$season > 2005)) return(pbp)
```

Comment:
This computes the minimum of a logical vector instead of the minimum season.

Reply:
Ok I think fixed